controllers/machinehealthcheck_controller.go

```diff
@@ -223,15 +223,27 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 			"max unhealthy", m.Spec.MaxUnhealthy,
 			"unhealthy targets", len(unhealthy),
 		)
+		message := fmt.Sprintf("Remediation restricted due to exceeded number of unhealthy machines (total: %v, unhealthy: %v, maxUnhealthy: %v)",
+			totalTargets,
+			m.Status.CurrentHealthy,
+			m.Spec.MaxUnhealthy,
+		)
+
+		// Remediation not allowed, the number of not started or unhealthy machines exceeds maxUnhealthy
```

Comment:
I think this comment text should be what we're using for the message text.  It's clearer than the existing message.

Reply:
Would you include the values still or omit those? 
Eg:
```
Remediation not allowed, the number of not started or unhealthy machines exceeds maxUnhealthy (total: 4, unhealthy: 3, maxUnhealthy: 40%)
```

Reply:
Yeah, the totals are fine, I'm just talking about the text part.